app/views/flood_risk_engine/enrollments/steps/_individual_address.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'flood_risk_engine/shared/address', form: form, step: step, f: f %>
```

Comment:
I am guessing other *_address.html.erb steps should use this DRY approach now? If so might be worth creating a github issue to make sure those changes are picked up?